dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -23,4 +23,43 @@ docPath=MicroProfile OpenAPI document endpoint path
 docPath.desc=Specifies the URL path for obtaining OpenAPI documents. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.).
 
 uiPath=MicroProfile OpenAPI UI endpoint path
-uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
\ No newline at end of file
+uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
+
+include=MicroProfile OpenAPI included modules.
+include=A comma separated list of modules that will be included in the OpenAPI document. Modules are in the format of {ApplicationName}/{ModuleName} or {ApplicationName} which refer to to <TODO, sort out the confusion of names>. An ApplicationName which does not specify a module will include all modules in the application. The special value "all" will include all available modules.
```

Comment:
Suggested changes to remove "will":

A comma-separated list of modules to include in the OpenAPI document. Modules are in the format of {ApplicationName}/{ModuleName} or {ApplicationName}, which refer to <TODO, sort out the confusion of names>. An ApplicationName that does not specify a module includes all modules in the application. The special value "all" includes all available modules.